CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 ## 0.1: It Starts with a Single Die (2015-07-25)
 
 - :game_die: Support for rolling individual dice.
-- :game_die: Initial support for expressions (e.g. `2d8 + 8`). Greatly expanded support planned for the next release.
+- :game_die: Initial support for expressions (e.g. `2d8 + 8`). Greatly expand support planned for the next release.
 - :memo: Initial README, LICENSE, CONTRIBUTING, and CHANGELOG.
+
+## 0.2: Dice all the way down
```

Comment:
Also, this whole section should be above 0.1. It's in reverse chronological order.